RadarSDK/RadarLocationManager.m

```diff
@@ -892,6 +1101,11 @@ - (void)locationManager:(CLLocationManager *)manager didEnterRegion:(CLRegion *)
         return;
     }
 
+    if ([region.identifier hasPrefix:kRampUpGeofenceIdentifierPrefix]) {
+
+        return;
+    }
+
```

Comment:
Hmmm, do we actually want this?

Guessing this was done to avoid the issue with notifications, which have since been handled